scss/variables/_variables.modular-scale.scss

```diff
@@ -4,7 +4,7 @@
 
 
 // Modular Scale - https://github.com/modularscale/modularscale-sass
-@import 'modularscale';
+@import '../../node_modules/dcf/scss/functions/modularscale/modularscale';
```

Comment:
This import is giving me an error. Can't find that stylesheet, and it doesn't look like it is in dcf either [/scss/functions](https://github.com/digitalcampusframework/dcf/tree/3.0/scss/functions)

Reply:
Oh wait I see that there is a PR [DCF#536](https://github.com/digitalcampusframework/dcf/pull/536)

Reply:
Looks good once I updated the DCF node module to that PR's branch